app/transfer/get-data-filename.js

```diff
@@ -11,7 +11,7 @@ const getDataFilename = (controlFilename, direction) => {
     return controlFilename.replace('.ctl', '')
   }
   if (controlFilename.endsWith('.ctl') && controlFilename.startsWith('GENESIS')) {
-    return controlFilename.replace('.ctl', '.gni')
+    return controlFilename.replace('.ctl', '')
```

Comment:
Think we can simplify this and combine with above if statement as both are now identical regardless of if inbound or not

Reply:
Good spot